src/executable.js

```diff
@@ -41,4 +41,16 @@ export class Executable extends mix(NamedDefaultableInMemoryEntity).with(Runtime
     getFlavorByConfig(config) {
         return config ? this.getFlavorByName(config.name) : this.defaultFlavor;
     }
+
+    getFlavorByApplicationVersion(version) {
```

Comment:
It returns an array of flavors, so we should name it `getFlavorsByApplicationVersion`